message/tests.py

```diff
@@ -0,0 +1,83 @@
+from django.utils import timezone
+from .models import Message
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import AuthorOptions
+import pytest
+
+AUTHOR = 'testuser@gmail.com'
+DOG_OWNER_ID = '123456789'
+DAYCARE_ID = '987654321'
+DATE = timezone.now()
+
+
+@pytest.fixture
+def dogowner_message1():
+    return Message.create(author=AuthorOptions.DogOwner,
+                          dogowner_id=DogOwner.objects.get(pk=1),
```

Comment:
Does this work outside of your dev environment? where do we take the dogowner and daycare entities from? I'm pretty sure you need to create those for the test.

Reply:
While that's fine, I'd suggest using a fixture in creating a specific test instance of the DogOwner to use for the tests.

Reply:
I wanted to maintain a uniform language, all the models in our project that require other models instances for test use the predefined one
None of them created an instance of a different model.

BTW in the first versions of the tests I created a dog owner then got this suggest
https://github.com/redhat-beyond/Hotails/pull/37#discussion_r841054825

Reply:
Again, it is fine, but not very readable and less flexible